### PR TITLE
- extended 'dmsetup table' parser to provide target and stripes

### PR DESCRIPTION
**Pull request by aschnell**

### CODE REVIEW

File: storage/SystemInfo/CmdDmsetup.h [hunk 1]
```diff
@@ -94,31 +94,38 @@ namespace storage
...
-	// TODO this will likely need to be extended to provide more table-data, e.g. the type
-
-	struct Entry
+	struct Table
 	{
-	    Entry() : majorminors() {}
+	    Table(const string& target)
+		: target(target), stripes(0), stripe_size(0), majorminors() {}
+
+	    string target;
```
Comment (mvidner) [inline storage/SystemInfo/CmdDmsetup.h]: Which values are valid for `target`?

Comment (aschnell) [inline storage/SystemInfo/CmdDmsetup.h]: I didn't find a simple and complete list, the best I know is https://github.com/torvalds/linux/tree/master/Documentation/device-mapper.

Comment (mvidner) [inline storage/SystemInfo/CmdDmsetup.h]: Oh. I thought there would be just a handful, in which case we could list them in a comment.

**Comment by mvidner** [comment]
LGTM.

**Pull request merged**